genesyscloud/conversations_messaging_integrations_open/data_source_genesyscloud_conversations_messaging_integrations_open_test.go

```diff
@@ -0,0 +1,90 @@
+package conversations_messaging_integrations_open
+
+import (
+	"fmt"
+	"testing"
+
+	cmMessagingSetting "terraform-provider-genesyscloud/genesyscloud/conversations_messaging_settings"
+	cmSupportedContent "terraform-provider-genesyscloud/genesyscloud/conversations_messaging_supportedcontent"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+/*
+Test Class for the conversations messaging integrations open Data Source
+*/
+
+func TestAccDataSourceConversationsMessagingIntegrationsOpen(t *testing.T) {
+	t.Parallel()
+	var (
+		dataSourceId                                    = "data_test_messaging_open"
+		resourceId                                      = "test_messaging_open"
```

Comment:
Could you change these to `dataSourceLabel` and `resourceLabel` ? We are changing our naming conventions 

More details here - https://inindca.atlassian.net/browse/DEVTOOLING-924